ros2param/ros2param/verb/load.py

```diff
@@ -34,8 +34,11 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
         parser.add_argument(
             '--include-hidden-nodes', action='store_true',
             help='Consider hidden nodes as well')
-        arg = parser.add_argument(
+        parser.add_argument(
             'parameter_file', help='Parameter file')
+        parser.add_argument(
+            '--use-wildcard', action='store_true',
+            help='Load parameters in the \'/**\' namespace into the node')
```

Comment:
The default behavior should be that parameters from the wildcard section are loaded.
If you want, there can be a `--no-use-wildcard` option

Reply:
Makes sense, please check the latest changes.